docs/customization/repository.rst

```diff
@@ -67,29 +78,40 @@ As we are selecting Products we need to have a join to translations, because the
 We are sorting the results by the count of how many products are still available on hand, which is saved on the ``onHand`` field on the specific ``variant`` of each product.
 Then we are limiting the query to 8 by default, to get only 8 products that are low in stock.
 
-**2.** In order to use your repository you need to configure it in the ``config/_sylius.yaml``.
+**2.** In order to use your repository you need to configure it in the ``config/packages/_sylius.yaml``.
+As you can see in the ``_sylius.yaml`` you already have a basic configuration, now you just need add your repository and override resourceRepository
 
 .. code-block:: yaml
 
     sylius_product:
         resources:
             product:
                 classes:
-                    repository: App\Repository\ProductRepository
+                    model: App\Entity\Product\Product
+                    repository: App\Repository\ProductRepository # you need add your repository to the config file
+                translation:
+                    classes:
+                        model: App\Entity\Product\ProductTranslation
```

Comment:
I guess that here you should add only a `repository: App\Repository\ProductRepository`